api-extractor.json

```diff
@@ -349,7 +349,7 @@
      */
     "tsdocMessageReporting": {
       "default": {
-        "logLevel": "error"
+        "logLevel": "none"
```

Comment:
The reason I had to do this is because I wanted to add the regex pattern to the comments of the Event Payload. But the api docs formatter does not like certain symbols. It's easier to just turn off that kind of error. No individual docs are actually generated for search object the subfields anyways.